api-put-object.go

```diff
@@ -40,6 +40,7 @@ type PutObjectOptions struct {
 	CacheControl       string
 	EncryptMaterials   encrypt.Materials
 	NumThreads         uint
+	StorageClass       string
```

Comment:
Need to update docs/API.md with this option.

Reply:
done